app/services/amr/csv_parser.rb

```diff
@@ -2,13 +2,13 @@
 
 module Amr
   class CsvParser
-    def initialize(config, file_name)
-      @file_name = file_name
+    def initialize(config, path_and_file_name)
+      @path_and_file_name = path_and_file_name
       @config = config
     end
 
     def perform
-      CSV.read("#{@config.local_bucket_path}/#{@file_name}", col_sep: @config.column_separator, row_sep: :auto)
+      CSV.read(@path_and_file_name, col_sep: @config.column_separator, row_sep: :auto)
```

Comment:
This allows us to parse temporary files which are not in the correct local bucket path